internal/common/baseclient/baseSpeaker.go

```diff
@@ -1,16 +1,18 @@
 package baseclient
 
 import (
+	"github.com/SOMAS2020/SOMAS2020/internal/common/gamestate"
 	"github.com/SOMAS2020/SOMAS2020/internal/common/rules"
 	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
 )
 
 type BaseSpeaker struct {
+	gameState gamestate.ClientGameState
```

Comment:
doesnt this need to be initialised in GetClientSpeakerPointer() in baseclient iigo

Reply:
Ah I knew I forgot smth, thanks, changed in latest commit